explorer/src/lib/components/detail-list/DetailList.svelte

```diff
@@ -0,0 +1,19 @@
+<svelte:options immutable={true} />
+
+<script>
+  import { setContext } from "svelte";
+  import contexts from "$lib/contexts";
+  import { makeClassName } from "$lib/dusk/string";
+  import "./DetailList.css";
+
+  /** @type {string | Undefined} */
+  export let className = undefined;
+
+  setContext("DUK:list-item:context", contexts.LIST_ITEM.DETAIL_LIST);
```

Comment:
Can you update `DUK` to `DUSK`?  DUK was leftover form the Dusk UI Kit, which is no longer in play.